src/components/Button/Button.stories.mdx

```diff
@@ -31,55 +31,203 @@ import { action } from "@storybook/addon-actions";
-        <Icon
-          name={args.iconType}
-          decorative={true}
-          modifiers={["small", "icon-left"]}
-        />
-      )}
-      Button Text
-      {!args.displayLeft && (
-        <Icon
-          name={args.iconType}
-          decorative={true}
-          modifiers={["small", "icon-right"]}
-        />
-      )}
+    }}
+  >
+    {args => (
+      <Button {...args}>
+        {args.displayLeft && (
+          <Icon
+            name={args.iconType}
+            decorative={true}
+            modifiers={["small", "icon-left"]}
+          />
+        )}
+        {args.buttonText}
+        {!args.displayLeft && (
+          <Icon
+            name={args.iconType}
+            decorative={true}
+            modifiers={["small", "icon-right"]}
+          />
+        )}
+      </Button>
+    )}
+  </Story>
+</Canvas>
+
+The icon can be placed to the left or the right of the button text.
+
+<Canvas>
+  <div
+    style={{
+      display: "flex",
+      justifyContent: "space-between",
+      width: "200px",
+    }}
+  >
+    <Button buttonType={ButtonTypes.Secondary}>
+      <Icon
+        name={IconNames.arrow}
+        decorative={true}
+        iconRotation={IconRotationTypes.rotate90}
+        modifiers={["small", "icon-left"]}
+      />
+      Previous
     </Button>
-  )}
-</Story>
+    <Button buttonType={ButtonTypes.Secondary}>
+      Next
+      <Icon
+        name={IconNames.arrow}
+        decorative={true}
+        iconRotation={IconRotationTypes.rotate270}
+        modifiers={["small", "icon-right"]}
+      />
+    </Button>
+  </div>
+</Canvas>
 
-<br />
+Text in a button is optional if an icon is rendered, but make sure that there
+is an `aria-label` passed to the `Button` component through the `attributes` prop:
+`attributes={{ "aria-label": "Previous" }}`
+
+<Canvas>
+  <div
+    style={{
+      display: "flex",
+      justifyContent: "space-between",
+      width: "200px",
+    }}
+  >
+    <Button
+      buttonType={ButtonTypes.Secondary}
+      attributes={{
+        "aria-label": "Previous",
+      }}
+    >
+      <Icon
+        name={IconNames.arrow}
+        decorative={true}
+        iconRotation={IconRotationTypes.rotate90}
+        modifiers={["small"]}
+      />
+    </Button>
+    <Button
+      buttonType={ButtonTypes.Secondary}
+      attributes={{
+        "aria-label": "Next",
+      }}
+    >
+      <Icon
+        name={IconNames.arrow}
+        decorative={true}
+        iconRotation={IconRotationTypes.rotate270}
+        modifiers={["small"]}
+      />
+    </Button>
+    <Button
+      buttonType={ButtonTypes.Secondary}
+      attributes={{
+        "aria-label": "Close",
+      }}
+    >
+      <Icon name={IconNames.close} decorative={true} modifiers={["small"]} />
+    </Button>
+  </div>
+</Canvas>
+
+## Patterns
+
+Buttons are blue by default but should be red when it is intended as a call out.
+The "Back to Top" button should have secondary button styles and an up arrow icon.
```

Comment:
The "Back to Top" button must be of type "Secondary" and must include an up arrow icon.